new scraper for sexselector.com & twistys.com, update multiple scrapers

aussiehuddo opened the pull request:
shout out to Drew and peolic on Discord for the assistance

peolic commented on scrapers/SexSelector.yml:
```diff
@@ -0,0 +1,35 @@
+        selector: //a[@class="header__menu header__menu--logo"]/@href
+        postProcess:
+          - replace:
+              - regex: .+(pg\d+)
+                with: $1/poster/${1}_01_2160.jpg
+              - regex: ^
+                with: "https://sm-members.bangbros.com/shoots/sexselector/"
```
didn't notice this on discord, but there's no reason not to have it be:
```suggestion
              - regex: .+(pg\d+)
                with: https://sm-members.bangbros.com/shoots/sexselector/$1/poster/${1}_01_2160.jpg
```

aussiehuddo commented on scrapers/BangBros.yml:
```diff
@@ -22,7 +22,12 @@ xPathScrapers:
                 with: "https:"
       Studio:
-        Name: //div[@class="vdoCast"]/a[1]/text()
+        Name: 
```
Added a map for momishorny to display correctly with spaces

aussiehuddo commented:
Had to move twistys.com to a separate scraper as the site layout has changed and no longer works with the MindGeek scraper.
Changes to other scrapers are fairly self-explanatory, fixing broken scraping and adding post-processing to get more accurate Studio Names.

aussiehuddo commented:
All changes for this PR:
- SCRAPERS-LIST.md - updated to include all changes
- Algolia_Adultime.yml - Added outofthefamily.com url
- Algolia_TabooHeat.yml - New scraper for tabooheat.com
- Algolia_xEmpire.yml - New scraper for xEmpire sites
- BangBros.yml - added studio name map for 'Mom is Horny'
- Brazzers.yml - Added babes.com url
- Dogfart.yml - Added studio name mapping
- DorcelClub.yml - updated studio mapping to match stash-box
- FreakMobMedia.yml - updated studio name from "FreakMob Media" to "Freak Mob Media" to match stash-box
- GammaEntertainment.yml - removed xEmpire sites due to new scraper
- hussiemodels.yml - updated studio mapping to match stash-box
- JulesJordan.yml - added gallery scraping and studio name mapping
- MindGeek.yml - removed twistys sites and added studio name mapping
- Mylf.yml - updated studio scraping and mapping to match stash-box ####
- Naughty America.yml - removed changes as already updated with other PR
- Nympho.yml - Fixed details fetching. now works for all sites.
- PaperStreetMedia.yml - updated studio mapping to match stash-box
- Penthouse.yml - removed changes as already updated with other PR
- ProducersFun.yml - updated details scraping to fix
- purgatoryx.yml - update studio scraping so that it now matches to the Heaven or Hell series
- RealityKingsOL.yml - removed babes.com url. Added Studio mapping to match stash-box
- SexSelector.yml - new scraper for sexselector.com
- TabooHeat.yml - deleted due to new scraper.
- ThirdRockEnt.yml - updated studio mapping to match stash-box
- trafficpimps.yml - updated studio mapping to match stash-box
- Twistys.yml - new scraper

@bnkai @Belleyy Can someone please give feedback on this as to how it can get committed. I apologise that I kept adding stuff to it as I am new to github. I can separate things out if that is easier.

bnkai commented:
@aussiehuddo the main issue is that all changes are in a single PR. That means that all files need to be checked before the PR is merged. I think I have checked most of the files ( the ones with the less changes are checked first  ) , I will try to finish the rest this weekend.

bnkai commented on scrapers/Brazzers.yml:
```diff
@@ -8,6 +8,7 @@ sceneByURL:
       - brazzers.com/video
       - men.com/sceneid/
       - realitykings.com/scene/
+      - babes.com/scene/
```
realitykings and babes.com no longer work with cdp/this scraper
Most of the Mindgeeks sites probably have to move to the MindGeekAPI scraper anyway

aussiehuddo replied:
removed realitykings.com and babes.com

aussiehuddo commented:
@bnkai updated

bnkai commented on scrapers/Mylf.yml:
```diff
@@ -33,9 +33,18 @@ xPathScrapers:
       Studio:
         Name:
-          selector: //div[contains(@class,"siteName")]/a[@href]/text()
+          selector: //div[contains(@class,"siteName")]/a[@href]/text()[2]
```
We can use
```suggestion
          selector: //div[contains(@class,"siteName")]/a[@href]
```
and let stash parse the inner text....

bnkai commented on scrapers/ProducersFun.yml:
```diff
@@ -8,7 +8,7 @@ xPathScrapers:
     scene:
       Title: //div[@class="shadow video-details"]/h1/text()
-      Details: (//div[@class="shadow video-details"]/p[not(@class)])[1]/text()
+      Details: (//div[@class="shadow video-details"]/p[not(@class)])[1]/text()[3]
```
Does not work with `https://producersfun.com/video/chanel-grey-a-fucking-conversation`
If there was an issue with the existing selector just use the below
```suggestion
      Details: (//div[@class="shadow video-details"]/p[not(@class)])[1]
```
We select the whole `<p>` and let stash get the inner text
